web.config.json

```diff
@@ -0,0 +1,54 @@
+{
+  "server": "https://<your-ocis-domain>",
+  "theme": "https://<your-ocis-domain>/themes/owncloud/theme.json",
+  "openIdConnect": {
+    "metadata_url": "https://<your-ocis-domain>/.well-known/openid-configuration",
+    "authority": "https://<your-ocis-domain>",
+    "client_id": "web",
+    "response_type": "code",
+    "scope": "openid profile email"
+  },
+  "options": {
+    "topCenterNotifications": true,
+    "disablePreviews": true,
+    "displayResourcesLazy": false,
+    "sidebar": {
+      "shares": {
+        "showAllOnLoad": true
+      }
+    },
+    "previewFileMimeTypes": [
+      "image/gif",
+      "image/png",
+      "image/jpeg",
+      "text/plain",
+      "image/tiff",
+      "image/bmp",
+      "image/x-ms-bmp"
+    ],
+    "contextHelpersReadMore": true
+  },
+  "apps": [
+    "files",
+    "text-editor",
+    "pdf-viewer",
+    "search",
+    "external",
+    "admin-settings",
+    "ocm",
+    "webfinger"
+  ],
+  "external_apps": [
+    {
+      "id": "preview",
+      "path": "web-app-preview",
+      "config": {
+        "mimeTypes": ["image/tiff", "image/bmp", "image/x-ms-bmp"]
+      }
+    },
+    {
+      "id": "presentation-viewer",
+      "path": "https://<url-where-the-dist-folder-is-served>/web-app-presentation-viewer.js"
+    }
+  ]
+}
```

Comment:
Are we adding this config file at the root of project?

Reply:
we can put it also somewhere else, but need to write the docs accordingly
where would you want to have it?